Source/JavaScriptCore/interpreter/Interpreter.cpp

```diff
@@ -516,6 +580,37 @@ void Interpreter::getStackTrace(JSCell* owner, Vector<StackFrame>& results, size
             return IterationStatus::Continue;
         }
 
+        if (Options::useAsyncStackTrace()) {
+            if (visitor->callee().isCell()) {
+                if (JSFunction* callee = jsDynamicCast<JSFunction*>(visitor->callee().asCell())) {
+                    JSGlobalObject* globalObject = callFrame->lexicalGlobalObject(vm);
+                    if (callee == globalObject->asyncFunctionResumeFunction()) {
+                        // When DFG optimization inlines calls to @asyncFunctionResume, the JSGenerator argument
+                        // position varies depending on the caller function defined in PromiseOperations.js and
+                        // AsyncFunctionPrototype.js. DFGByteCodeParser performs inlining via inlineCall(), and
+                        // DFGStackLayoutPhase remaps argument positions using stackOffset accumulation.
```

Comment:
But this is not guaranteed right? If the caller is also inlining this function too, then these arguments are unrelated.